examples/networking/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 set(EXAMPLE_FILES simulation.py)
-set(EXAMPLE_PROGRAMS simple_tc.py tc_perf_event.py)
+set(EXAMPLE_PROGRAMS simple_tc.py tc_perf_event.py net_monitor.py simulation.py sockmap.py)
```

Comment:
`simulation.py` is added above and it does not contain any BPF program.

Reply:
ops sorry. done here https://github.com/iovisor/bcc/pull/4312/commits/cc661d2dee6ddc4814fb504a76b72a14a5e02daa